pallets/tx-pause/src/lib.rs

```diff
@@ -103,25 +123,12 @@ pub mod pallet {
             pallet_name: Vec<u8>,
             function_name: Vec<u8>,
```

Comment:
does it really make sense to want to pause individual extrinsics?

Reply:
individual extrinsic pause/unpause is first ported from acala. As we've `pause_transactions` for multiple extrinsics, we could remove individual extrinsic pause/unpause. But I feel it's better to keep it for simply in some case we just want to pause/unpause one single extrinsic. Anyway, it's open for discussion.

Reply:
i meant to ask why we pause extrinsics and not pallets.
Is there any pallet where we want to pause some extrinsics and not others?
I cannot think of of one.
So i'd say pausing `pallets` is good enough and we could use either pallet names or pallet IDs.
This would help address @ghzlatarev 's question if we could avoid using `String`s